fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java

```diff
@@ -140,8 +141,13 @@ public void contextInitialized(ServletContextEvent event) {
      */
     private void bootstrapDerbyDatabases(PropertyGroup fhirConfig) throws Exception {
         Boolean performDbBootstrap = fhirConfig.getBooleanProperty(PROPERTY_JDBC_BOOTSTRAP_DB, Boolean.FALSE);
+        // Controls if we run derby in debugging mode which enables more logs.
+        boolean DEBUG = false;
```

Comment:
it's not really a constant at this point. let's just pass in the boolean value and a comment on line 149....

Reply:
agreed, thanks!

Reply:
done